src/lib/utils.py

```diff
@@ -0,0 +1,209 @@
+import os
+
+from os import path
+from os.path import join as opj
+import pandas as pd
+from typing import List
+
+
+def participants_tsv():
+
+    participants_tsv = opj(directories()["exp"], "participants.tsv")
+
+    return pd.read_csv(participants_tsv, sep="\t")
+
+
+def load_config():
+
+    config_file = opj(
+        path.dirname(path.realpath(__file__)), "analysis_pipelines_config.tsv"
+    )
+
+    return pd.read_csv(config_file, sep="\t")
+
+
+def return_team_config(team_ID: str) -> dict:
+    """
+    Args:
+        team_ID (str):
+
+    Returns:
+        dict: configuration for analysis for this team
+
+    Partly read from ``src/lib/analysis_pipelines_config.tsv``
+
+    keys:
+
+    - ``teamID``
+    - ``n_participants``
+    - ``excluded_participants``
+    - ``func_fwhm``
+    - ``directories`` see the ``directories`` function
+    - ``subject_list``: List[str] of the subject labels after filtering
+      the ``excluded_participants``
+
+    """
```

Comment:
will grow in content but having a dictionary that lists as much as possibke the specification of analysis pipeline of each team is likely to help avoid duplication in code